src/query-builder/relation-id/RelationIdLoader.ts

```diff
@@ -45,18 +45,18 @@ export class RelationIdLoader {
                             const duplicateParts: Array<string> = []
                             relationIdAttr.relation.joinColumns.forEach(
                                 (joinColumn) => {
-                                    result[joinColumn.databaseName] =
-                                        this.connection.driver.prepareHydratedValue(
-                                            rawEntity[
-                                                DriverUtils.buildAlias(
-                                                    this.connection.driver,
-                                                    undefined,
-                                                    relationIdAttr.parentAlias,
-                                                    joinColumn.databaseName,
-                                                )
-                                            ],
-                                            joinColumn.referencedColumn!,
-                                        )
+                                    const columnAlias = DriverUtils.buildAlias(
```

Comment:
This entire file can be reverted if https://github.com/tursodatabase/libsql-client-ts/pull/166 gets merged.